modules/command-not-found/init.zsh

```diff
@@ -13,8 +13,19 @@ if [[ -s '/etc/zsh_command_not_found' ]]; then
 elif [[ -s '/usr/share/doc/pkgfile/command-not-found.zsh' ]]; then
   source '/usr/share/doc/pkgfile/command-not-found.zsh'
 # Load command-not-found on macOS when homebrew tap is configured.
-elif [[ -s '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-command-not-found/handler.sh' ]]; then
-  source '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-command-not-found/handler.sh'
+elif (( $+commands[brew] )) \
+        && [[ -s "${cnf_command::=$(brew command command-not-found-init 2> /dev/null)}" ]]; then
```

Comment:
This is the main downside right here - this call is painfully slow, taking almost a full second on my computer... and there isn't really a way to turn it off or cache.

Reply:
Yeah. It's fine if it's slow when I "miss" on a command... but not fine on every terminal instantiation...

Reply:
Wow, ~1s just for `brew command` is rough and rather strange.
Running `time (brew command command-not-found-init)` gives me around 0.4s - 0.5s on a very slow non-retina MBP.